src/commands/importWebApp/importWebApp.ts

```diff
@@ -0,0 +1,321 @@
+                }
+                ext.outputChannel.appendLine(localize("importWebApp", "Import Web App succeed!"));
+            }
+        ).then(async () => {
+            // tslint:disable-next-line:no-non-null-assertion
+            await node!.refresh();
+            window.showInformationMessage(localize("importWebApp", `Imported Web App '${webAppName}' to API Management succesfully.`));
+        });
+    }
+}
+
+export async function getPickedWebApp(node: ApiTreeItem | ApisTreeItem, webAppType: string): Promise<Site> {
+    const client = azureClientUtil.getClient(node.root.credentials, node.root.subscriptionId, node.root.environment);
+    const allfunctionApps = await listWebApps(client, webAppType);
+    return await pickWebApp(allfunctionApps);
+}
+
+export async function listWebApps(client: WebSiteManagementClient, webAppType: string): Promise<Site[]> {
+    const allWebApps: WebAppCollection = await client.webApps.list();
+    if ((webAppType === "webApp")) {
+        return allWebApps.filter((ele) => !!ele.kind && !ele.kind.includes("functionapp"));
+    }
+    return allWebApps.filter((ele) => !!ele.kind && ele.kind.includes("functionapp"));
+}
+
+export async function pickWebApp(apiApps: Site[]): Promise<Site> {
+    // Pick function app
+    const apiApp = await ext.ui.showQuickPick(apiApps.map((s) => { return { label: nonNullOrEmptyValue(s.name), site: s }; }), { canPickMany: false });
+    return apiApp.site;
+}
+
+// Create policy for importing
+export function createImportXmlPolicy(backendId: string): string {
+    const operationPolicy = [{
+        policies: [
+            {
+                inbound: [
+                    { base: null },
+                    {
+                        "set-backend-service": [
+                            {
+                                _attr: {
+                                    id: "apim-generated-policy",
+                                    "backend-id": backendId
+                                }
+                            }
+                        ]
+                    }
+                ]
+            },
+            { backend: [{ base: null }] },
+            { outbound: [{ base: null }] },
+            { "on-error": [{ base: null }] }
+        ]
+    }];
+
+    return xml(operationPolicy);
+}
+
+// Create backend for the web app
+export async function setAppBackendEntity(node: ApisTreeItem | ApiTreeItem, backendId: string, appName: string, appPath: string, appResourceGroup: string, webAppName: string, BackendCredentials?: BackendCredentialsContract): Promise<void> {
+    const nbackend: BackendContract = {
+        description: `${appName}`,
+        resourceId: `${node.root.environment.resourceManagerEndpointUrl}/subscriptions/${node.root.subscriptionId}/resourceGroups/${appResourceGroup}/providers/Microsoft.Web/sites/${webAppName}`,
```

Comment:
create a function for getWebAppResourceId() or getSiteResourceId()  `${node.root.environment.resourceManagerEndpointUrl}/subscriptions/${node.root.subscriptionId}/resourceGroups/${appResourceGroup}/providers/Microsoft.Web/sites/${webAppName}`